PlayTools/PlayLoader.m

```diff
@@ -32,17 +32,21 @@
 void *my_dyld_get_base_platform(void *platform) { return 2; }
 
 // #define DEVICE_MODEL ("iPad13,8")
-#define DEVICE_MODEL ("iPad8,6")
+//#define DEVICE_MODEL ("iPad8,6")
 
 // find Mac by using sysctl of HW_TARGET
 // #define OEM_ID ("J522AP")
-#define OEM_ID ("J320xAP")
+// #define OEM_ID ("J320xAP")
+
+// get device model from playcover .plist
+#define DEVICE_MODEL ([[[PlaySettings shared] GET_IPAD_MODEL] UTF8String])
+#define OEM_ID ([[[PlaySettings shared] GET_OEM_ID] UTF8String])
 
 static int my_uname(struct utsname *uts) {
   int result = 0;
-  NSString *nickname = @"ipad";
-//   NSString *productType = @"iPad13,8";
-  NSString *productType = @"iPad8,6";
+    NSString *nickname = @"ipad";
+  //   NSString *productType = @"iPad13,8";
+    NSString *productType =  [[PlaySettings shared] GET_IPAD_MODEL];
```

Comment:
Does this `NSString` really needed?
Looks strncpy with `DEVICE_MODEL` can do same thing (at line 54)

Reply:
> Does this `NSString *productType` really needed? Looks strncpy with `DEVICE_MODEL` can do same thing (at line 54)
@KhoraLee 
It was like that before, i know can just use DEVICE_MODEL, but otherwise it crashes for some reason.

Reply:
I was asking because `strncpy(uts->machine, DEVICE_MODEL, strlen(DEVICE_MODEL) + 1);` runs without any issue to me.
<img width="910" alt="image" src="https://user-images.githubusercontent.com/18005062/184273570-e37b4730-8d03-4d86-bfc0-2979104424f6.png">

Reply:
Mhh Let me try again

Reply:
Ok, It worked now